graviti/utility/lazy.py

```diff
@@ -233,6 +238,26 @@ def create_list(
         self._lists.append(lazy_list)
         return lazy_list
 
+    def get_lazylists(
+        self, extractors: Mapping[str, Union[_Extractor, _Extractors]]
```

Comment:
can we refine these typehint? Is it not 100% correct right now?

Reply:
Mapping cannot be replaced by Dict, what else could be refined? use overload?

Reply:
What I mean is not replace the Mapping with Dict.
I mean the input extractors is a nested `_Extractor`, it may have multilayer of dict, then the last value is `_Extractor`.
But the type hint here is just one layer.

Reply:
there are some hint:
```python
class _Type(Protocol):
    def __getitem__(self, key: str) -> Union["_Type", int]:
        ...
```
But it is your job to search and find out the solution.